typescript/phoenix-sdk/src/orderPacket.ts

```diff
@@ -243,17 +242,17 @@ export function getImmediateOrCancelOrderPacket({
   lastValidUnixTimestampInSeconds,
 }: {
   side: Side;
-  priceInTicks?: number;
+  priceInTicks: number | null;
```

Comment:
remove null here (from interface) and coerce it only when the COption thing is required